ofl/playpensans/article/ARTICLE.en_us.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+tr, th, td {
+    border-bottom: 1px dotted #d2d2d2;
+    padding: 0.3rem;
+    text-align: center;
+}
+td {
+    font-size: 40px;
+}
+</style>
+</head>
+<body>
```

Comment:
I think everything above that line should be removed, may come in conflict with the website.

Reply:
Yeah, was hesitant, but was worth trying it.